go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/coreos/go-oidc v2.2.1+incompatible
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/go-redis/redis/v7 v7.4.0
+	github.com/go-redis/redis/v8 v8.0.0
```

Comment:
Is this change required in the update to the new Go version as a dependency somewhere?

Otherwise can we move this to its own PR to split up the scope of these changes?